tests/fixtures/issues/issue-1156.php

```diff
@@ -1,59 +1,6 @@
 <?php
 
 return [
-    'issue-1156-1' => [
```

Comment:
I removed this test as it doesn't really have a proper FB string in it (it's missing the FBAV piece).  It's also missing device info, which is suspect.